src/main/java/pl/durilian/wordTermsChecker/entities/Exam.java

```diff
@@ -10,6 +10,8 @@
 @Getter
 @Setter
 public class Exam {
+    static ConfigurationManager configurationManager = ConfigurationManager.getInstance();
```

Comment:
Jeśli ta zmienna jest wykorzystana tylko w jednej metodzie to powinna być lokalna w tej metodzie